packages/select/src/Select.story.tsx

```diff
@@ -39,6 +40,7 @@ storiesOf('Select', module)
             name="pets"
             defaultValue="cat"
             disabled={disabled}
+            usePortal={usePortal}
```

Comment:
This isn't something we really need to add as a knob.

Reply:
I just figured it made it easier for y'all to test locally, but I am happy to remove if you'd like.

Reply:
It does, and I suppose it doesn't matter too much since we're moving to the website soon! If we were intending to keep using storybook, I'd ask we remove this.

Reply:
Sounds good